tests/test_benchmark_simple_scenarios.py

```diff
@@ -99,23 +99,40 @@ def test_benchmark_ABE_grid_pv_bat(self, margs):
             input_type=CSV_EXT,
             path_output_folder=os.path.join(TEST_OUTPUT_PATH, use_case),
         )
-        with open(
-            os.path.join(TEST_OUTPUT_PATH, use_case, "json_with_results.json")
-        ) as fp:
-            AB_dict_values = convert_special_types(json.load(fp))
+        #with open(
+            #os.path.join(TEST_OUTPUT_PATH, use_case, "json_with_results.json")
+        #) as fp:
+            #AB_dict_values = convert_special_types(json.load(fp))
+
+        AB_busses_flow = pd.read_excel(
+            os.path.join(TEST_OUTPUT_PATH, use_case, "timeseries_all_busses.xlsx"),
+            sheet_name="Electricity bus",
+        )
+        #AB_busses_flow = AB_busses_flow.set_index("Unnamed: 0")
+        # compute the sum of the excess electricity
+        AB_busses_flow["Electricity excess_sink"] = AB_busses_flow.sum()
 
         use_case = "ABE"
         main(
             path_input_folder=os.path.join(TEST_INPUT_PATH, use_case),
             input_type=CSV_EXT,
             path_output_folder=os.path.join(TEST_OUTPUT_PATH, use_case),
         )
-        with open(
-            os.path.join(TEST_OUTPUT_PATH, use_case, "json_with_results.json")
-        ) as fp:
-            ABE_dict_values = convert_special_types(json.load(fp))
+        #with open(
+            #os.path.join(TEST_OUTPUT_PATH, use_case, "json_with_results.json")
+        #) as fp:
+            #ABE_dict_values = convert_special_types(json.load(fp))
+
+        ABE_busses_flow = pd.read_excel(
```

Comment:
This could be written in a loop over (usecase1, usecase2) as most of the code is copy pasted. At each loop end, you can collect the value of interest and store it in a dict (let's name it d for the sake of simplicity) which keys could be (usecase1, usecase2), the test would then be:
d[usecase1] > d[usecase2]